src/cow-react/modules/limitOrders/containers/SettingsWidget/index.tsx

```diff
@@ -51,7 +48,12 @@ export function SettingsWidget() {
             </styledEl.ExpertModeIndicator>
           )}
         </styledEl.SettingsButton>
-      </Dropdown>
+        <styledEl.MenuContent>
+          <MenuItem disabled={true} onSelect={() => void 0}>
```

Comment:
Is this `onSelect` intentional?

Reply:
<img width="597" alt="image" src="https://user-images.githubusercontent.com/7122625/222678679-bfa0c866-82ed-4d37-94bf-153532e39aca.png">

`@reach/menu-button` requires `onSelect`, can't avoid this